pos_mail/static/src/js/pos.js

```diff
@@ -56,11 +70,11 @@ odoo.define('pos_mail.pos', function (require) {
 
         send_mail_receipt: function(partner_id) {
             var receipt = QWeb.render('PosMailTicket', this.get_receipt_render_env());
-            var order_name = this.pos.get_order().name;
+            var order = this.pos.get_order().name;
```

Comment:
what was the reason to change?
old name much more representable than the new one